### ISSUE TITLE
android 11 y 12 open camera

### ISSUE THREAD

**Issue by mane06** [mgks/Android-SmartWebView#244]
Hello I would like to know if there are plans to implement the camera opening in android 11 and 12

**Comment by mgks** [comment]
yes, there are plans to solve this but due to insufficient resources and time limitations it wasn't solved yet, hopefully it will be fixed in a month or two, for the current ones and the upcoming Android release.

**Comment by mgks** [comment]
Fixed with #318
